src/plugins/plot/src/configuration/PlotSeries.js

```diff
@@ -160,7 +160,7 @@ define([
                     var newPoints = _(this.data)
                         .concat(points)
                         .sortBy(this.getXVal)
-                        .uniq(true, this.getXVal)
+                        .uniq(true, point => [this.getXVal(point), point.eu].join())
```

Comment:
`point.eu` is VISTA specific. It's also not the only attribute that users might plot. `point.getYVal()` is a more generic way to retrieve the Y value corresponding to a given X for the currently selected Y axis.

Reply:
ahh yes good catch, thanks

Reply:
fixed in https://github.com/nasa/openmct/pull/2515/commits/0eff81991308869d4ec14b86fcd837c82cba111c